src/net_processing.h

```diff
@@ -93,7 +93,7 @@ class PeerManager : public CValidationInterface, public NetEventsInterface
     virtual void SendPings() = 0;
 
     /** Set the best height */
-    virtual void SetBestHeight(int height) = 0;
+    virtual void SetBestBlock(int height, uint64_t time) = 0;
```

Comment:
Maybe change the argument from `uint64_t` to `std::chrono::seconds`, so that the units of it are more clear. The comment warrants an update:

```cpp
/** Set the height of the best block and its time (seconds since epoch). */
virtual void SetBestBlock(int height, std::chrono::seconds time) = 0;
```

Reply:
Done as suggested